schemas/json/theme.json

```diff
@@ -253,6 +253,20 @@
 							"type": "boolean",
 							"default": true
 						},
+						"fluid": {
+							"description": "Enables fluid typography and allows users to set global fluid typography parameters.",
+							"type": "object",
+							"properties": {
+								"maxViewportWidth": {
```

Comment:
Can we call this just max / min?

Reply:
I'll run with `typography.fluid.max|min` 

I suppose the schema JSON documents the props. I'll also add doc notes just in case there is any ambiguity about their application.